src/components/klarnaExpressCheckoutButton/KlarnaExpressCheckoutButton.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+
+class KlarnaCreditPromotionBadge extends React.Component {
```

Comment:
Should we rename this class to `KlarnaExpressCheckoutButton`?

Reply:
Ah yes, thank you! :sweat_smile: